data/sql/updates/pending_db_world/rev_1554420147566038300.sql

```diff
@@ -0,0 +1,122 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1554420147566038300');
+
+-- Gelkis Windchaser SAI
+SET @ENTRY := 4649;
+UPDATE `creature_template` SET `AIName`="SmartAI" WHERE `entry`=@ENTRY;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(@ENTRY,0,0,0,0,0,100,0,500,1000,3000,4500,11,9532,32,0,0,0,0,2,0,0,0,0,0,0,0,'Gelkis Windchaser - In Combat - Cast \'Lightning Bolt\''),
+(@ENTRY,0,1,0,2,0,100,0,10,35,15000,20000,11,959,32,0,0,0,0,1,0,0,0,0,0,0,0,'Gelkis Windchaser - Between 10-35% Health - Cast \'Healing Wave\''),
+(@ENTRY,0,2,0,2,0,100,1,1,10,0,0,25,0,0,0,0,0,0,1,0,0,0,0,0,0,0,'Gelkis Windchaser - Between 1-10% Health - Flee For Assist (No Repeat)');
+
+-- Gelkis Stamper SAI
+SET @ENTRY := 4648;
+UPDATE `creature_template` SET `AIName`="SmartAI" WHERE `entry`=@ENTRY;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(@ENTRY,0,0,0,9,0,100,0,10,1,3500,5000,11,5568,0,0,0,0,0,2,0,0,0,0,0,0,0,'Gelkis Stamper - Within 10-1 Range - Cast \'Trample\''),
+(@ENTRY,0,1,0,2,0,100,1,1,10,0,0,25,0,0,0,0,0,0,1,0,0,0,0,0,0,0,'Gelkis Stamper - Between 1-10% Health - Flee For Assist (No Repeat)');
+
+-- Gelkis Scout SAI
+SET @ENTRY := 4647;
+UPDATE `creature_template` SET `AIName`="SmartAI" WHERE `entry`=@ENTRY;
```

Comment:
" e.e I saw that one! Same on the others, if u don't care, i'll fix it :)

Reply:
Dammit! I'm sorry I'm sleepy... If you will 😅